src/Services/src/CfCli/CfCliService.cs

```diff
@@ -98,6 +99,23 @@ public async Task<Version> GetApiVersion()
             }
         }
 
+        /// <summary>
+        /// Retrieves an access token to be used for authentication with Cloud Foundry UAA.
+        /// <para>
+        /// A cached value of the access token will be returned as long as the expiration time of the token has not yet been reached.
+        /// If there is no cached value for the access token or the expiration time of the cached token has been reached, this method 
+        /// will invoke the CF CLI oauth-token command (<see cref="_getOAuthTokenCmd"/>) to acquire a new access token. Invoking 
```

Comment:
General thoughts on what gets documented.  The "see cref" targets an internal scoped var.  In general, API docs are for API clients who typically cannot access internal/private things.  If, however, these docs are for team members, then doc'ing private/internal stuff is cool.

Reply:
Ok yeah this same thought crossed my mind in the moment -- first of all, I don't think it's crucial to include this `see cref` at all (I think the summary would be just fine without it), so I'm happy to get rid of it if it's unconventional.

These docs are mainly just for our own benefit at the moment, but if an external client did want to use this method, what exactly would the downside be with documenting an internal variable like this? Would it just seem pointless if that value is inaccessible?

Reply:
Correct.